app/views/setup/_polling.html.slim

```diff
@@ -0,0 +1,4 @@
+= content_for :page_javascript do
+  javascript:
```

Comment:
We removed turbolinks in portus for other reasons (e.g. nginx couldn't pick it up in a containerized environment). Since we are not using it (and we might move into React or Vue), I'm ok with removing turbolinks.